src/mainField/ButtonTreat.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+
+export const ButtonTreat = ({
+  stateIndicator,
+  setStateIndicator,
+  setButtonFeedEnabled,
+  buttonTreatDisabled,
+  setButtonTreatDisabled,
+  setIsRecovered,
+}) => {
+  useEffect(() => {
+    if (stateIndicator === "sick") {
+      setButtonTreatDisabled(false);
+    } else if (stateIndicator === "dead") {
+      setButtonTreatDisabled(true);
+    }
+  }, [stateIndicator]);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setStateIndicator("fine");
+    setButtonFeedEnabled("false");
```

Comment:
why use a string where a boolean should be used?

Reply:
fixed